pkg/operator/config.go

```diff
@@ -7,13 +7,15 @@ import (
 	"path/filepath"
 
 	configv1 "github.com/openshift/api/config/v1"
+	"k8s.io/apimachinery/pkg/util/sets"
 )
 
 const (
 	// TODO(alberto): move to "quay.io/openshift/origin-kubemark-machine-controllers:v4.0.0" once available
 	clusterAPIControllerKubemark = "docker.io/gofed/kubemark-machine-controllers:v1.0"
 	clusterAPIControllerNoOp     = "no-op"
 	kubemarkPlatform             = configv1.PlatformType("kubemark")
+	MAPOFeature                  = "MachineAPIProviderOpenStack"
```

Comment:
Does this need to be exported? Is there a const within the OpenShift API repo we can use instead?

Reply:
Its stored as a string key in json, the const just helps keep the lookup consistent.